src/get_target_data.py

```diff
@@ -485,13 +489,24 @@ def test_target_data():
     # time series row count should = 5 days * 3 clades * 52 locations
     assert ts.height == 5 * 3 * 52
 
-    expected_time_series_cols = set(["location", "target_date", "clade", "observation"])
+    expected_time_series_cols = set(
+        [
+            "location",
+            "target_date",
+            "clade",
+            "observation",
+            "nowcast_date",
+            "sequence_as_of",
+        ]
+    )
     assert set(ts.columns) == expected_time_series_cols
 
     assert set(ts.get_column("clade").to_list()) == {"AA", "BB", "other"}
     assert ts.get_column("target_date").min() == date(2024, 11, 30)
     assert ts.get_column("target_date").max() == date(2024, 12, 4)
     assert ts.get_column("observation").sum() == 20
+    assert ts.get_column("nowcast_date").unique().to_list() == ["2024-9-11"]
```

Comment:
@elray1 Do you think it's also worth adding a `tree_as_of` column here, so anyone use this file outside the context of the hub can more easily understand which reference tree was used to get the clade assignments?

Reply:
It seems like that could potentially be useful, though not necessarily high priority.  I don't really have strong feelings, but I guess it could be nice to do that.

Reply:
@elray1 ok, stuck `tree_as_of` in there